website/source/api/system/sealwrap-rewrap.html.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "api"
+page_title: "/sys/sealwrap/rewrap - HTTP API"
+sidebar_title: "<code>/sys/sealwrap/rewrap</code>"
+sidebar_current: "api-http-system-sealwrap-rewrap"
+description: |-
+  The `/sys/sealwrap/rewrap` endpoint is used to rewrap all seal wrapped entries.
+---
+
+# `/sys/sealwrap/rewrap`
+
+~> **Enterprise Only** – These endpoints require Vault Enterprise.
+
+The `/sys/sealwrap/rewrap` endpoint is used to rewrap all seal wrapped entries.
```

Comment:
Would be nice to include why someone might want to use a rewrap. e.g. after a seal migration, if you rotate the remote keyring and want to upgrade entries

Reply:
Added in #7689